mbuild/formats/hoomd_simulation.py

```diff
@@ -38,6 +38,7 @@ def create_hoomd_simulation(
     pppm_kwargs={"Nx": 8, "Ny": 8, "Nz": 8, "order": 4},
     init_snap=None,
     restart=None,
+    nlist="cell",
```

Comment:
Since we require hoomd to be imported here anyway, would it be more useful to be able to pass around the neighborlist object? Im not entirely sure what would be best for you and other users. What do you think @jennyfothergill ?

Reply:
Yeah, I'm fine with that change. Probably the getattr method has more room for error, but I do like the simplicity of passing in a string name.

Reply:
That is as close as you can get for hoomd v2 typing. 

In v3 we expose the base neighbor list class as a documented type.

Reply:
OK, thank you!